### PR TITLE
feat: Adding the new overlay read me

### LINKED ISSUE #8326: Selects including derivative functions return noncontinous timestamp sequences and ORDER BY time doesn't improve the situation
System info: 

Linux version 3.2.0-4-amd64 (debian-kernel@lists.debian.org) (gcc version 4.6.3 (Debian 4.6.3-14) ) #1 SMP Debian 3.2.68-1+deb7u3

InfluxDB Server: v1.2.2

Steps to reproduce:

1. Issue a select on a measurement containing counter values and utilizing the derivative or non_negative_derivative function. Example:

```
SELECT derivative("writes", 1s) FROM "diskio" WHERE "host" = 'fastnetmon.neterra.net' AND "name" = 'sda'  AND time > now() - 15m

diskio
time	                                derivative
...
2017-04-26T15:53:10Z	10.8
2017-04-26T15:53:20Z	19.6
2017-04-26T15:53:30Z	22.1
2017-04-26T15:53:40Z	37
2017-04-26T15:53:50Z	11
2017-04-26T15:54:00Z	17.2
2017-04-26T15:54:10Z	10
2017-04-26T15:54:20Z	14.4
2017-04-26T15:54:30Z	16.6
2017-04-26T15:54:40Z	10.6
2017-04-26T15:40:00Z	15.70340909090909   <------ Discontinutie
2017-04-26T15:40:10Z	22.8
2017-04-26T15:40:20Z	27.2
2017-04-26T15:40:30Z	17.7
2017-04-26T15:40:40Z	11.7
2017-04-26T15:40:50Z	9.9
...
```

At the same time the raw data has correct sequence of timestamps:

```
SELECT "writes" FROM "diskio" WHERE "host" = 'fastnetmon.neterra.net' AND "name" = 'sda'  AND time > now() - 15m

diskio
time	                                writes
...
2017-04-26T15:53:10Z	61184231
2017-04-26T15:53:20Z	61184427
2017-04-26T15:53:30Z	61184648
2017-04-26T15:53:40Z	61185018
2017-04-26T15:53:50Z	61185128
2017-04-26T15:54:00Z	61185300
2017-04-26T15:54:10Z	61185400
2017-04-26T15:54:20Z	61185544
2017-04-26T15:54:30Z	61185710
2017-04-26T15:54:40Z	61185816
2017-04-26T15:54:50Z	61186046
2017-04-26T15:55:00Z	61186155
2017-04-26T15:55:10Z	61186309
2017-04-26T15:55:20Z	61186455
2017-04-26T15:55:30Z	61186549
2017-04-26T15:55:40Z	61186687
...
```

Expected behavior: 

Before the upgrade to 1.2.2, while using the 1.1.0 rc  version  the derivatives returned normal, continuous time sequences results.

Actual behavior: 
The results are not continuous in time and appear overlapped in Grafana:

Additional info: 

Here is a screenshot from Grafana, illustrating the effects:

screenshot-192 168 7 126-3000-2017-04-26-18-49-54

Using the "ORDER BY time [DESC]"  doesn't seem to do any better.

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #8326

This will have a read me open when you press the community template button

- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

File: ui/cypress/e2e/communityTemplates.test.ts [hunk 1]
```diff
@@ -226,15 +226,9 @@ describe('Community Templates', () => {
...
-    it('takes you to github when you click on the Community Templates link', () => {
-      cy.getByTestID('template-source-link').within(() => {
-        cy.contains('Community Templates').should(
-          'have.attr',
-          'href',
-          'https://github.com/influxdata/community-templates/blob/master/docker/docker.yml'
-        )
-      })
-      //TODO: add the link from CLI
+    it('takes you to github readme when you click on the Community Templates button', () => {
```
Comment (hoorayimhelping) [inline ui/cypress/e2e/communityTemplates.test.ts:229]: we'll want to keep an eye on this. these tests actually make GET requests to github, which might be against TOS if we do it in an automated test.

File: ui/src/templates/components/CommunityTemplateReadMeOverlay.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  Props,
+  State
+> {
+  state = {isVisible: false}
+
+  render() {
+    return (
+      <React.Fragment>
+        <Button
+          text="Community Templates"
```
Comment (hoorayimhelping) [inline ui/src/templates/components/CommunityTemplateReadMeOverlay.tsx:24]:
```suggestion
          text="View Readme"
```

File: ui/src/templates/components/CommunityTemplateReadMeOverlay.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          icon={IconFont.GitHub}
+          size={ComponentSize.Small}
+          onClick={this.showOverlay}
+          testID="community-template-readme-overlay-button"
+          style={{display: 'inline-block'}}
+        />
+        <Overlay visible={this.state.isVisible}>
+          <Overlay.Container maxWidth={800} testID="template-install-overlay">
```
Comment (hoorayimhelping) [inline ui/src/templates/components/CommunityTemplateReadMeOverlay.tsx:32]: this `800` is also used elsewhere in templates. might be time to make it a named constant so we don't have some [magic numbers](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) hanging around

File: ui/cypress/e2e/communityTemplates.test.ts (hunk 1 above)
Comment (zoesteinkamp) [reply ui/cypress/e2e/communityTemplates.test.ts:229]: good to know, i did not know about Githubs TOS

**Pull request merged**